README.md

```diff
@@ -312,7 +312,7 @@ Please [discover modm's peripheral drivers for your specific device][discover].
 <td align="center">○</td>
 <td align="center">○</td>
 <td align="center">✗</td>
-<td align="center">✗</td>
+<td align="center">○</td>
```

Comment:
This pops in, because the RF chips have the following xml tag:
```
<module name="FLASH">
  <instance name="FLASH" caption="FLASH Controller">
    <register-group name="FLASH" name-in-module="FLASH" offset="0x00" address-space="data" caption="FLASH Controller"/>
  </instance>
</module>
```

I don't see anything useful here and I guess this wasn't the intent of this field. Should I remove these lines via patch?

Reply:
Yes, but you should remove it in the Python code to be generic: https://github.com/modm-io/modm-devices/blob/develop/tools/generator/dfg/avr/avr_device_tree.py#L74-L88

Reply:
Fixed.